.gitignore

```diff
@@ -7,23 +7,20 @@
 *.wast.hpp
 *.dot
 *.abi.hpp
-*.cmake
-!.cicd
-!package.cmake
-!CMakeModules/*.cmake
 *.ninja
 \#*
 \.#*
 CMakeCache.txt
 CMakeFiles
-cmake_install.cmake
 cmake-build-debug/
 cmake-build-release/
```

Comment:
Can we change these to `cmake-build-*/` please. I have for example `cmake-build-debug-clang11/`.